ports/unix/mpbthciport.c

```diff
@@ -140,10 +144,10 @@ STATIC int configure_uart(void) {
 
     // Enable HW RTS/CTS flow control.
     toptions.c_iflag &= ~(IXON | IXOFF | IXANY);
-    toptions.c_cflag |= CRTSCTS;
+    toptions.c_cflag &= ~CRTSCTS;
 
     // 1Mbit (TODO: make this configurable).
-    speed_t brate = B1000000;
+    speed_t brate = B115200;
```

Comment:
Why did this and the above CRTSCTS setting change?

Reply:
That was accidentally left in from an experiment with the Pico W issues. Fixed.